File: IOS 2018/NextStep/NextStep/ViewController.m
//
//  ViewController.m
//  NextStep
//
//  Created by baimi on 2018/8/15.
//  Copyright © 2018年 liubang. All rights reserved.
//

#import "ViewController.h"

#import "AFNetworking.h"

@interface ViewController ()


@end

@implementation ViewController
- (IBAction)oushhome:(id)sender {
    
}

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [self showdata];
    
    
    
    
    // Do any additional setup after loading the view, typically from a nib.
}
-(void)showdata{
    AFHTTPSessionManager * session  = [AFHTTPSessionManager manager];
    NSDictionary *dict = @{@"foo":@"bar"};
    [session POST:@"https://httpbin.org/post" parameters:dict progress:^(NSProgress * _Nonnull downloadProgress) {
        
    } success:^(NSURLSessionDataTask * _Nonnull task, id  _Nullable responseObject) {
        NSLog(@"请求成功");
        NSLog(@"-----%@ -----%@",responseObject ,responseObject[@"origin"]);
        
    } failure:^(NSURLSessionDataTask * _Nullable task, NSError * _Nonnull error) {
        NSLog(@"请求失败");
    }];
    
    //
   
    
    
    
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: IOS 2018/swift--helloworld/swift--helloworld/ViewController.swift
//
//  ViewController.swift
//  swift--helloworld
//
//  Created by baimi on 2018/8/6.
//  Copyright © 2018年 liubang. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let rect = CGRect(x: 0, y: 100, width: SCREEN_WIDTH, height: 55)
        
        let lbColor = gof_RGBColor(33,44,55)
        
        let label = UILabel(frame:rect)
        label.backgroundColor = lbColor
        label.textAlignment = .center
        label.text = "什么鬼"
        label.font = UIFont(name:"Arial", size: 36)
        label.textColor = UIColor.red
        
        self.view.addSubview(label)
        
        
        
        
        
        
        
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

